test/e2e/framework/util.go

```diff
@@ -4491,7 +4479,34 @@ func CheckConnectivityToHost(f *Framework, nodeName, podName, host string, timeo
 			Logf("pod %s/%s logs:\n%s", f.Namespace.Name, pod.Name, logs)
 		}
 	}
+	return err
+}
 
+// CheckConnectivityToHost launches a pod to test connectivity to the specified
+// host. An error will be returned if the host is not reachable from the pod.
+//
+// An empty nodeName will use the scheduler to choose where the pod is executed.
+func CheckConnectivityToHost(f *Framework, nodeName, host string, timeout int) error {
+
+	podName := "ping-test"
+	cmdPing := []string{
```

Comment:
@oomichi Should we just remove the ping?

Reply:
@timothysc 
I guess it would work fine in most cases/environments by just removing the ping test.
However I feel it is safe to keep the test to avoid failing in few environments. Maybe I am a little conservative.